stl/inc/xstring

```diff
@@ -1223,7 +1223,7 @@ public:
     constexpr basic_string_view(const basic_string_view&) noexcept            = default;
     constexpr basic_string_view& operator=(const basic_string_view&) noexcept = default;
 
-    /* implicit */ constexpr basic_string_view(_In_z_ const const_pointer _Ntcts) noexcept // strengthened
+    /* implicit */ constexpr basic_string_view(const const_pointer _Ntcts) noexcept // strengthened

```

Comment:
Could we instead suppress the warning by wrapping the entire class template definition a la:
```c++
#pragma warning(push)
// Invalid annotation: 'NullTerminated' property may only be used on buffers whose elements are of integral or pointer type
#pragma warning(disable : 6510)

template <class _Elem, class _Traits>
class basic_string_view {
    /* ... */
};

#pragma warning(pop)
```

Reply:
Yes, it seems it works: https://gcc.godbolt.org/z/ac81raaaW
Thanks!